### DISCUSSION TITLE
Urgent: Account Suspension After Following GitHub Education Guidelines – Help Needed!

### DISCUSSION THREAD

**Discussion by codesBySaad** [community/community#135610] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello GitHub Community and Support Team,

I’m reaching out with an urgent issue regarding the suspension of my GitHub account (https://github.com/itxSaaad).

### Background:

- I recently re-applied for the GitHub Student Developer Pack, which was approved. However, I didn’t receive access to some resources like GitHub Copilot even after three days.
- Following the instructions provided in the GitHub Education documentation that is also not showing up now but was there on application portal where will get access to the **benefits soon** the benefits soon was a  link to the docs, I raised a discussion to address the delay in accessing the resources. My Discussion Link: [https://github.com/orgs/community/discussions/135537]
- A few hours later, when I tried to log in to my account, I found that it had been suspended without any prior warning or explanation.

### Why This Is Critical:
My GitHub account holds over three years of coding history, including:

1. Personal projects
2. Client work
3. Job-related repositories

This data is extremely important to me, and I believe the suspension may have been a misunderstanding as I was merely following the guidelines provided.

### What I’ve Done So Far:
I’ve submitted a reinstatement request (Ticket ID: 2936367) and am awaiting a response.
I haven’t received any communication explaining the suspension, and I’m growing increasingly concerned about losing access to my critical data.

### Request for Help:
If anyone from the GitHub Support Team or the community can assist in expediting the review of my case, I would be extremely grateful. This account is vital to my work and personal projects, and I need urgent help to get it reinstated.

Thank you for your attention and any support you can provide.

Best regards,
Muhammad Saad Faisal

Note: Ive Created This new Account for the Discussion and Recovery of my Previous Account

**Comment by LiteBrite82** [comment]
Hi @codesBySaad 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

**Comment by codesBySaad** [comment]
How much time its gonna take as it contains important repos?

**Comment by codesBySaad** [comment]
Also i created an organization: [Zironix Limited](https://github.com/Zironix-Limited/) with that account and was only owner of that organization how will i get access to that organization? will it be abandoned? @LiteBrite82

**Comment by codesBySaad** [comment]
I added a new Comment on the discussion please check that out..
